packages/browser-integration-tests/suites/public-api/startTransaction/basic_usage/subject.js

```diff
@@ -1,30 +1,35 @@
-const transaction = Sentry.startTransaction({ name: 'test_transaction_1' });
-const span_1 = transaction.startChild({
-  op: 'span_1',
-  data: {
-    foo: 'bar',
-    baz: [1, 2, 3],
-  },
-});
-for (let i = 0; i < 2000; i++);
+async function run() {
+  const transaction = Sentry.startTransaction({ name: 'test_transaction_1' });
+  const span_1 = transaction.startChild({
+    op: 'span_1',
+    data: {
+      foo: 'bar',
+      baz: [1, 2, 3],
+    },
+  });
 
-// span_1 finishes
-span_1.finish();
+  await new Promise(resolve => setTimeout(resolve, 1));
 
-// span_2 doesn't finish
-const span_2 = transaction.startChild({ op: 'span_2' });
-for (let i = 0; i < 4000; i++);
+  // span_1 finishes
+  span_1.finish();
 
-const span_3 = transaction.startChild({ op: 'span_3' });
-for (let i = 0; i < 4000; i++);
+  // span_2 doesn't finish
+  const span_2 = transaction.startChild({ op: 'span_2' });
+  await new Promise(resolve => setTimeout(resolve, 1));
 
-// span_4 is the child of span_3 but doesn't finish.
-const span_4 = span_3.startChild({ op: 'span_4', data: { qux: 'quux' } });
+  const span_3 = transaction.startChild({ op: 'span_3' });
+  await new Promise(resolve => setTimeout(resolve, 1));
 
-// span_5 is another child of span_3 but finishes.
-const span_5 = span_3.startChild({ op: 'span_5' }).finish();
+  // span_4 is the child of span_3 but doesn't finish.
+  const span_4 = span_3.startChild({ op: 'span_4', data: { qux: 'quux' } });
 
-// span_3 also finishes
-span_3.finish();
+  // span_5 is another child of span_3 but finishes.
+  const span_5 = span_3.startChild({ op: 'span_5' }).finish();
 
-transaction.finish();
+  // span_3 also finishes
+  span_3.finish();
+
+  transaction.finish();
+}
+
+run();
```

Comment:
are we not awaiting this on purpose?

Reply:
there is nothing to await here, this just runs on init and there is no top-level await I think there :)